app/src/main/res/values/strings.xml

```diff
@@ -252,4 +252,11 @@ be temporary, and you can try again later.</li>
     <string name="notification_request_desktop_site">Requested desktop version</string>
     <!-- Notification displayed when non-desktop version of the site is requested-->
     <string name="notification_request_non_desktop_site">Returned to non-desktop version</string>
+
+    <!-- Notification displayed to introduce a "YouTube casting" feature (similar to "broadcasting")
+    Users who have the YouTube app on their phone or tablet can use it to play YouTube videos directly on their tv.
+
+    %s will be replaced with the icon that matches the button in the YouTube app for this feature.
```

Comment:
Can we inline an icon? Does it need to be a unicode character to do that?

Reply:
We can use ImageSpan for that.